src/FSharpPlus/Extensions/Array.fs

```diff
@@ -101,3 +101,12 @@ module Array =
     /// <returns>Array with corresponding pairs of input arrays.</returns>
     let zipShortest (a1: array<'T1>) (a2: array<'T2>) =
         Array.init (min a1.Length a2.Length) (fun i -> a1.[i], a2.[i])
+
+    /// <summary>Applies given function to each value of the given Array.</summary>
```

Comment:
Can we add a more accurate description? This looks like `map`.  Worst case "Choose with access to the index." is more appropriate than this.

Reply:
improved all summaries of choosei in extensions